product/models.py

```diff
@@ -0,0 +1,90 @@
+from django.db import models
+
+#1.카테고리테이블
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'Categories'
```

Comment:
```suggestion
        db_table = 'categories'
```

테이블 이름은 클래스 명과 달리 모두 소문자로 사용합니다. 띄어쓰기는 언더바 `_`로 구분합니다. 아래 모든 클래스에 적용해주세요